### PR TITLE
write logs to std without log PV mounted

### PR DESCRIPTION
**Pull request by MegaByte875**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

### CODE REVIEW

**Review by wey-gu — APPROVED** [review]
🎉

File: apis/apps/v1alpha1/nebulacluster_common.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ const (
 	AgentInitContainerName    = "ng-init-agent"
 	DefaultAgentPortGRPC      = 8888
 	agentPortNameGRPC         = "grpc"
-	defaultAgentImage         = "megabyte875/nebula-agent:latest"
+	defaultAgentImage         = "vesoft/nebula-agent:latest"
```
Comment (wenhaocs) [inline apis/apps/v1alpha1/nebulacluster_common.go:45]: why was `megabyte875/nebula-agent:latest` previously?

File: apis/apps/v1alpha1/nebulacluster_metad.go [hunk 2]
```diff
@@ -101,10 +101,16 @@ func (c *metadComponent) GetDataStorageClass() *string {
...
 
 func (c *metadComponent) GetLogStorageResources() *corev1.ResourceRequirements {
+	if c.nc.Spec.Metad.LogVolumeClaim == nil {
+		return nil
+	}
 	return c.nc.Spec.Metad.LogVolumeClaim.Resources.DeepCopy()
 }
 
 func (c *metadComponent) GetDataStorageResources() (*corev1.ResourceRequirements, error) {
+	if c.nc.Spec.Metad.DataVolumeClaim == nil {
```
Comment (wenhaocs) [inline apis/apps/v1alpha1/nebulacluster_metad.go:111]: I understand that the request is `LogVolumeClaim` can be nil. But why `DataVolumeClaim` can be nil too?

Comment (MegaByte875) [reply apis/apps/v1alpha1/nebulacluster_metad.go:111]: prevent panic when the DataVolumeClaim does not exist

**Pull request merged**